epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -86,6 +94,28 @@ private void cancelAnimation(LottieAnimationView lottieView) {
   }
 
   @Override
+  public boolean canAnimateChanges(ColorHolder view) {
+    return false;
+  }
+
+  @Override
+  public Animator animateChanges(ColorHolder view) {
+    int currentColor = ((ColorDrawable) view.getBackground()).getColor();
```

Comment:
Speaking of which, I think we should change the Lottie animation in the example as it might be confusing regarding the purpose of `bindChange`. Clicking the view doesn't actually change the state of the item so it should be handled by the view in a simple onClick-listener independent of all RV code. You wouldn't animate button hovering or something like that as a "state change" either. Instead we could use something like the heart animation in the Lottie samples that animates between two discrete states (undliked/liked):

![](https://github.com/airbnb/lottie-android/blob/master/gifs/Example1.gif?raw=true)